Firmware/MotorControl/commands.c

```diff
@@ -137,6 +138,7 @@ void motor_parse_cmd(uint8_t* buffer, int len, SerialPrintf_t response_interface
     // TODO very hacky way of terminating sscanf at end of buffer:
     // We should do some proper struct packing instead of using sscanf altogether
     buffer[len-1] = 0;
+    buffer++;
```

Comment:
Why skipping first byte?

Reply:
The first byte is always a $ now.  Previously, we stripped that framing byte, but since Gcode doesn't have framing bytes, it made sense to pass them to this layer.

Reply:
Thanks. Adding a comment about that would be good for future readers.